README.md

```diff
@@ -96,7 +93,12 @@ Minifier even has its own CLI that allows you quickly minify files in your termi
 
 ### Installation
 
-To install the CLI for minifier, run the following command: `deno install --allow-read --allow-write --allow-hrtime -n minifier https://deno.land/x/minifier/cli.ts`.
+To install the CLI for minifier, run the following command:
+
+```sh
+deno install --allow-read --allow-write --allow-hrtime \
```

Comment:
The newline is breaking the command in my terminal.  Do you think you could put the code block using single ticks again but still spaced out from the other text?

`deno install --allow-read --allow-write --allow-hrtime -n minifier https://deno.land/x/minifier/cli.ts`